wiki/Tournaments/oPHN/2023_taiko/en.md

```diff
@@ -0,0 +1,337 @@
+| **Yron8** ::{ flag=PH }:: | **6** | 1 | ::{ flag=PH }:: Krishel17 | [#1](https://osu.ppy.sh/community/matches/109957535) |
+| xidorn ::{ flag=PH }:: | 2 | **6** | ::{ flag=PH }:: **wwowie** | [#1](https://osu.ppy.sh/community/matches/109955171) |
+| **Mochii_31** ::{ flag=PH }:: | **6** | 3 | ::{ flag=PH }:: [KN]CuB-03 | [#1](https://osu.ppy.sh/community/matches/109954644) |
+
+### Round of 16
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **space-walker** ::{ flag=PH }:: | **5** | 2 | ::{ flag=PH }:: Taikore | [#1](https://osu.ppy.sh/community/matches/109881036) |
+| **Krishel17** ::{ flag=PH }:: | **5** | 3 | ::{ flag=PH }:: Euanofthedead | [#1](https://osu.ppy.sh/community/matches/109866894) |
+| **Kotaro** ::{ flag=PH }:: | **5** | 3 | ::{ flag=PH }:: xidorn | [#1](https://osu.ppy.sh/community/matches/109863284) |
+| **[KN]CuB-03** ::{ flag=PH }:: | **5** | 0 | ::{ flag=PH }:: -Birds- | *win by default* |
+| **COLE9001** ::{ flag=PH }:: | **5** | 0 | ::{ flag=PH }:: wwowie | [#1](https://osu.ppy.sh/community/matches/109850248) |
+| **Yron8** ::{ flag=PH }:: | **5** | 0 | ::{ flag=PH }:: Toadette | [#1](https://osu.ppy.sh/community/matches/109848590) |
+| **Mochii_31** ::{ flag=PH }:: | **5** | 0 | ::{ flag=PH }:: BPChingu | [#1](https://osu.ppy.sh/community/matches/109847994) |
+| **Farmer Brown** ::{ flag=PH }:: | **5** | 1 | ::{ flag=PH }:: Zyphore | [#1](https://osu.ppy.sh/community/matches/109849392) |
+
+## Ruleset
+
+### General rules
+
+1. This is an osu!taiko, double-elimination, 1v1 tournament open for players globally ranked from #500 to #10000 (inclusive) with the Philippine flag on their profile. Filipino players without the Philippine flag on their profile may present a proof of their Philippine citizenship in order to join
+2. Hosts, poolers, custom mappers, and playtesters are strictly not allowed to play. Also, players are allowed to staff as referee, streamer, or commentator once they are eliminated from the tournament.
+3. Matches will use ScoreV2 and NoFail in Head-to-Head mode.
+4. Players and staff must be treated with respect. Instructions from the hosts and referees are to be followed. Abusing the rules and poor sportsmanship may result in disqualification from the entire tournament.
+5. Any modification from the ruleset will be announced. Players must be active on the tournament's discord server in the case of these modifications.
+
+### Mappool structure
+
+- There will be separate mappools for each stage. Each mappool contains the following mod brackets: NoMod, Hidden, HardRock, DoubleTime, ForceMod, and Tiebreaker. There will be no Tiebreaker for Qualifiers.
+  - DoubleTime can be played as Nightcore if all players in the lobby agree to it.
+  - ForceMod will be played either with Hidden, HardRock, or Hidden + HardRock.
+  - Tiebreaker will be played with NoMod and will be only played in case of a tie.
```

Comment:
```suggestion
- There will be separate mappools for each stage. Each mappool contains the following mod brackets: No Mod, Hidden, Hard Rock, Double Time, Force Mod, and Tiebreaker. There will be no Tiebreaker for Qualifiers.
  - Double Time can be played as Nightcore if all players in the lobby agree to it.
  - Force Mod will be played either with Hidden, Hard Rock, or Hidden + Hard Rock.
  - Tiebreaker will be played with No Mod and will be only played in case of a tie.
```